contracts/src/starknet/rewaave/token_bridge.cairo

```diff
@@ -140,6 +142,29 @@ func initiate_withdraw{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_c
     return ()
 end
 
+@external
+func bridge_rewards{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+    l1_token : felt, l1_recipient : felt, amount : Uint256):
+    let (token_owner) = get_caller_address()
+
+    let (bridge_address) = get_contract_address()
+
+    let (reward_token) = IETHStaticAToken.get_REWARD_TOKEN(contract_address=l1_token)
+
+    # BURN REWARD TOKEN
+    IERC20.transferFrom(contract_address=reward_token, sender=token_owner, recipient=bridge_address, amount=amount)
```

Comment:
shouldn't we burn the reward token here?

Reply:
This change will be overriden (fix) when merged with your changes to token bridge